piton/tools/src/sims/sims,2.0

```diff
@@ -2825,12 +2830,13 @@ sub parse_args
   $ENV{PITON_ARIANE}   = $opt{ariane};
   $ENV{PITON_BLACKPARROT} = $opt{blackparrot};
   $ENV{PITON_RV64_PLATFORM}     = $opt{rv64_platform};
-  $ENV{BASEJUMP_STL_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/external/basejump_stl";
-  $ENV{BP_COMMON_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/bp_common";
-  $ENV{BP_FE_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/bp_fe";
-  $ENV{BP_ME_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/bp_me";
-  $ENV{BP_BE_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/bp_be";
-  $ENV{BP_TOP_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/bp_top";
+  $ENV{BASEJUMP_STL_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/black-parrot/external/basejump_stl";
+  $ENV{BP_COMMON_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/black-parrot/bp_common";
+  $ENV{BP_FE_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/black-parrot/bp_fe";
+  $ENV{BP_ME_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/black-parrot/bp_me";
+  $ENV{BP_BE_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/black-parrot/bp_be";
+  $ENV{BP_TOP_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/black-parrot/bp_top";
+  $ENV{HARDFLOAT_DIR} = $ENV{DV_ROOT} . "/design/chip/tile/blackparrot/black-parrot/external/HardFloat";
```

Comment:
Could we set these in piton_settings.bash instead maybe? Or have a blackparrot_setup.sh?